awslimits/manage.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sendgrid
+from flask_script import Manager
+
+from server import app
+from support import get_limits_for_alert, alert_email_body
+
+manager = Manager(app)
+
+
+@manager.command
+def send_alerts():
+	limits_for_alert = get_limits_for_alert()
+	if limits_for_alert:
+		content = alert_email_body(limits_for_alert)
+		email_recipients = [{'email': email} for email in os.environ.get('EMAIL_RECIPIENTS').split(',')]
+		sg = sendgrid.SendGridAPIClient(apikey=os.environ.get('SENDGRID_API_KEY'))
```

Comment:
Sorry, I wasn't clear. I am asking if we would work if we don't have the env var declared. Does it?